contents/docs/tutorials/b2b.md

```diff
@@ -0,0 +1,247 @@
+---
+title: Tracking Key B2B Product Metrics
+sidebar: Docs
+showTitle: true
+---
+
+<br />
+
+<small class="note-block centered">_Estimated Reading Time: 12 minutes ☕☕☕_</small>
+
+<br />
+
+<span class="larger-image">
+
+![B2B Tutorial Banner Image](../../images/tutorials/banners/b2b.png)
+
+</span>
+
+<br />
+
+If you run a B2B business, it is especially important to track group usage of your product, since, by definition, you have multiple clearly-defined groups as your clients. 
+
+Since usage may vary from company to company in your clients list, digging deeper into how these different teams and businesses use your product is essential to developing a platform that meets the use-case of your core audience well.
+
+As such, this tutorial will walk you through how to track usage of your B2B product, going over key metrics and how you can set up your analytics views to reflect them.
+
+### Pre-Requisites
+
+To follow this tutorial along, you need to:
+
+1. Have [deployed PostHog](/docs/deployment).
+1. Have started receiving events via our [snippet](/docs/integrations/js-integration), one of our [integrations](/docs/integrations), or our [API](/docs/api/overview)
+
+> **Pro Tip:** To make the most out of this tutorial, we recommend that you have some [actions](/docs/tutorials/actions#sorting-through-your-events-with-actions) set up, as well as have tried sending some [custom events](/docs/tutorials/actions#using-custom-events-to-track-advanced-behaviors). These are explained in detail in our [Complete Guide to Event Tracking](/docs/tutorials/actions). Additionally, it is also beneficial to be familiar with [Cohorts](/docs/tutorials/cohorts).
+ 
+
+### Determining Product-Market Fit
+
+Determining product-market fit is a key step to making your product successful, and, in order to do so, you need to understand if your product is being used, who is using it, and how it's being used. 
+
+#### Is my product being used?
+
+To determine if your product is being used, let's start in 'Trends' (under 'Insights').
+
+##### How many people use my product?
```

Comment:
'people or companies'? (then adjust below content to include how to get number of companies)